src/js/components/award/shared/activity/ContractGrantsActivityChart.jsx

```diff
@@ -164,6 +283,36 @@ const ContractGrantsActivityChart = ({
                     scale={yScale}
                     ticks={yTicks}
                     textAnchor="left" />
+                <ActivityXAxis
+                    height={height}
+                    width={visualizationWidth - padding.left}
+                    padding={padding}
```

Comment:
Why are we passing padding as props... Shouldn't this be like in dataMapping or something?

Reply:
We usually use them to calculate values in the components. We could refactor the way we create these visualizations.